openmdao/core/group.py

```diff
@@ -874,9 +874,10 @@ def _top_level_post_sizes(self):
                                            f"'{abs_out}' to serial input '{abs_in}' without "
                                            "specifying src_indices.")
                 elif in_dist and not out_dist:
-                    warn_deprecation(f"Connection between serial output '{abs_out}' and distributed"
-                                     f" input '{abs_in}' is deprecated and will become an error "
-                                     "in a future release.")
+                    issue_warning(f"Connecting a serial output '{abs_out}' and distributed input "
```

Comment:
What exactly are we asking the user to do here?  If I were a user and saw this message I would be worried that something was wrong but would have no idea what the 'expected' connections would be.  I wonder if we should even warn the user at all if we're not going to provide them with any useful information.